src/main/kotlin/com/pubnub/internal/v2/subscription/SubscriptionImpl.kt

```diff
@@ -22,6 +28,41 @@ internal class SubscriptionImpl(
     options: SubscriptionOptions? = null,
 ) : Subscription(), EventEmitter {
 
+    override var onMessage: ((PubNub, PNMessageResult) -> Unit)? = null
```

Comment:
same comment as SubSetImpl: use delegated properties

Reply:
ok, I will change it.